src/robusta/core/sinks/pagerduty/pagerduty_sink.py

```diff
@@ -48,14 +50,112 @@ def __to_pagerduty_status_type(title: str):
         else:
             return "trigger"
 
-    def write_finding(self, finding: Finding, platform_enabled: bool):
+    @staticmethod
+    def __send_changes_to_pagerduty(self, finding: Finding, platform_enabled: bool):
         custom_details: dict = {}
+        links = []
+        if platform_enabled:
+            href = finding.get_investigate_uri(self.account_id, self.cluster_name)
+            links.append({
+                "text": "🔂 See change history in Robusta",
+                "href": href
+            })
+        else:
+            links.append({
+                "text": "🔂 Enable Robusta UI to see change history",
+                "href": "https://bit.ly/robusta-ui-pager-duty"
+            })
+
+        source = self.cluster_name
+
+        custom_details["namespace"] = finding.service.namespace
+        custom_details["resource"] = f"{finding.service.resource_type}/{finding.subject.name}"
+
+        if finding.subject.node:
+            custom_details["node"] = finding.subject.node
+
+        timestamp = finding.starts_at.astimezone().isoformat()
+
+        for enrichment in finding.enrichments:
+            for block in enrichment.blocks:
+                changes = self.__block_to_changes(block, enrichment)
+                if not changes:
+                    continue
+
+                operation = changes["operation"]
+                if not operation:
+                    continue
+
+                unformatted_texts = self.__to_unformatted_text_for_changes(block)
+                if unformatted_texts:
+                    change_num = 1
+                    for diff_text in unformatted_texts:
+                        custom_details[f"Change {change_num}"] = diff_text
+                        change_num += 1
+
+                description = finding.description
+                changes_count_text = ""
+
+                if operation == K8sOperationType.UPDATE:
```

Comment:
`if operation == K8sOperationType.UPDATE and description:`

Reply:
Done